javaslang/src/test/java/javaslang/AbstractValueTest.java

```diff
@@ -981,14 +981,14 @@ private boolean isSerializable() {
                         instanceOf(Either.LeftProjection.class),
                         instanceOf(Either.RightProjection.class),
                         instanceOf(Future.class),
-                        instanceOf(Iterator.class),
-                        instanceOf(Validation.class)
+                        instanceOf(Iterator.class)
                 ), false),
                 Case(anyOf(
                         instanceOf(Either.class),
                         instanceOf(Option.class),
                         instanceOf(Try.class),
-                        instanceOf(Traversable.class)
+                        instanceOf(Traversable.class),
```

Comment:
In 3.0.0 Iterator will not implement Traversable any more (i.e. all internal `ofAll(iterator)` calls will be substituted with `ofAll(() -> iterator)`). So all Traversables will be Serializable.